### ISSUE TITLE
package.json references a version of cordova-ios that doesn't exist

### ISSUE THREAD

**Issue by geoidesic** [katzer/cordova-plugin-local-notifications#2050]
**WARNING: IF YOU IGNORE THIS TEMPLATE, WE'LL IGNORE YOUR ISSUE. YOU MUST FILL THIS IN!**

https://github.com/katzer/cordova-plugin-local-notifications/blob/7ea561a4eaa5cb061339b24d20e703625eda949e/package.json#L30

This line indicates that the minimum supported version of cordova-ios is v10. However, the current version of cordova-ios that's available is 7.1.1. There isn't a newer version on their repository. What gives?

## Your Environment

* Plugin version: 1.0.0
* Platform: Mac OS X
* OS version: 15.1
* Device manufacturer / model: Macbook Pro M1
* Cordova version (```cordova -v```): 12.0.0 (cordova-lib@12.0.1)
* Cordova platform version (```cordova platform ls```):  ios 7.1.1

## Expected Behavior

Ideally it should be installable.

## Actual Behavior

Gives an Unmet project requirements warning and then installs an old beta version

## Steps to Reproduce

1. cordova plugin add cordova-plugin-local-notification

## Context

Install the plugin

## Debug logs

```
% cordova plugin add cordova-plugin-local-notification
Unmet project requirements for latest version of cordova-plugin-local-notification:
    cordova-ios (7.1.1 in project, >=10.0.0 required)
Fetching highest version of cordova-plugin-local-notification that this project supports: 0.9.0-beta.2 (latest is 1.0.0)
```

**Comment by GitToTheHub** [comment]
Hi,

thanks for opening this issue. I corrected the incorrect dependency in `package.json` in https://github.com/katzer/cordova-plugin-local-notifications/commit/39d79c7b4b588c803888a96acf6b6eda8a2c3d19.

I will make an update on npm.

Regards,

Manuel

**Comment by GitToTheHub** [comment]
Hi, the package was update on npm and also released a new version `1.1.0`.

Regards
